content/02.introduction.md

```diff
@@ -51,20 +51,38 @@ Phylogenetic analysis of a PCR amplicon fragment from five patients along with t
+Human SARS-CoV-2 is genetically closer to bat RATG13 than to any other known virus, including human SARS-CoV. The similarity between SARS-CoV-2 and RATG13 is as high as 96.2%, while the similarity between SARS-CoV-2 and SARS-CoV is only 79% [@doi:10.1038/s41586-020-2012-7; @doi:10.1093/nsr/nwaa036].
+Nevertheless, some fragments between SARS-CoV-2 and RATG13 have a difference of up to 17%, suggesting a complex natural selection process during zoonotic transfer. 
+While the S fragment is highly similar to that of viruses found in pangolins [@doi:10.1101/2020.02.19.950253], there is no consensus about S origin in SARS-CoV-2.
+Potentially it can be a result either of recombination or coevolution [@doi:10.1093/nsr/nwaa036; @doi:10.1126/sciadv.abb9153].
+Anyway, while there is substantial support of the zoonotic origin of SARS-CoV-2, there is a lot of unknown, causing issues regarding the SARS-CoV-2 evolutionary pattern [@doi:10.1038/s41591-020-0820-9, @doi:10.1038/s41586-020-2012-7].
+
+After zoonotic transfer, SARS-CoV-2 continues evolving in the human population [@doi:10.1186/s12967-020-02344-6].
+Luckily, the SARS-CoV-2 mutation rate is moderate compared to other RNA viruses [@doi:10.1016/j.meegid.2020.104351], and that restricts SARS-CoV-2 evolution pace. 
+Nevertheless, genomic data show some significant statistical evidence of ongoing evolution.
+For example, the Spike G614 has already become dominant over D614 by June 2020 [@doi:10.1016/j.cell.2020.06.043].
+While there is a hypothesis that this genomic change increased the SARS-CoV-2 infectivity and virulence, there is no data yet to support or discard this hypothesis reliably [@doi:10.1016/j.cell.2020.06.040].
+The study [@doi:10.1016/j.meegid.2020.104351] has reported more than 198 recurrent mutations using a dataset of 7666 curated sequences showing the dynamic of SARS-CoV-2 evolution.
+Overall, while it is obvious that SARS-CoV-2 exhibits moderate potential to further evolution, there is much uncertainty about SARS-CoV-2 evolution trends and consequences.
+
+The numerous SARS-CoV-2 mutations [@doi:10.1186/s12967-020-02344-6; @doi:10.1093/nsr/nwaa036; @doi:10.1016/j.meegid.2020.104351], observed in SARS-CoV-2 individual strains, can be used as signals during outbreak investigations and discovering of transmission patterns.
+The sequencing data can be found at GISAID [@https://www.gisaid.org/], NCBI [@https://www.ncbi.nlm.nih.gov/sars-cov-2/], and COVID-19 data portal [@https://www.covid19dataportal.org/].
+The databases contain more than 75,000 strains sampled and sequenced from the patients from December 2019 to July 2020.
+This data helped determine the source of local COVID-19 outbreaks in Connecticut, [@doi:10.1016/j.cell.2020.04.021], New Your City area [@doi:10.1126/science.abc1917], and Iceland [@doi:10.1056/NEJMoa2006100].
```

Comment:
```suggestion
This data helped determine the source of local COVID-19 outbreaks in Connecticut (USA), [@doi:10.1016/j.cell.2020.04.021], the New Your City area (USA) [@doi:10.1126/science.abc1917], and Iceland [@doi:10.1056/NEJMoa2006100].
```
Proposal to be slightly less US centric, although we are unfortunately pretty US centric throughout the manuscript!

Reply:
Thanks!